app/views/questions/index.html.erb

```diff
@@ -32,6 +32,7 @@
           <h4 class="panel-title">
             <div class="row valign">
               <div class="col-sm-11">
+                <!-- Make this link to a modal, not a new page -->
```

Comment:
This change can be removed from this feature branch, it is relevant to a different feature. (see below for a suggestion of how to do this -- or just delete the line and then commit that change)